### DISCUSSION TITLE
Enhanced Battery Modelling

### DISCUSSION THREAD

**Discussion by BenWinchester** [CLOVER-energy/CLOVER#173] [IAA]
### Issue
Page 36 (section 5.1.1) of Version 1 of the User Guide for CLOVER 4.0 talks about the battery modelling not taking account of temperature effects and reduced cycling.

### Proposal
More advanced modelling within the Battery Module would be an enhancement to CLOVER, and, potentially, could be gradually implemented via additions to the Battery Module that are introduced incrementally and each have little impact on the runtime/performance of CLOVER.

**Comment by BenWinchester** [comment]
The battery degradation model could be improved to include degradation of static "full" batteries which aren't used.

**Comment by hamishbeath** [comment]
A possible route could be to have a simple temperature dependent degradation factor per unit of energy that passes through the battery. For example, 5 degrees above the recommended temp (such as 25 degrees C), degradation is 10% higher.
